rpa-builder/6.7/modules/ROOT/pages/_partials/generate-oauth-token-steps.adoc

```diff
@@ -0,0 +1,18 @@
+To generate and configure a refresh token:
+
+. In the *Tools* menu, select *Generate OAuth Refresh Token*.
++
+image:generate-oauth-token.png[The Generate OAuth Refresh Token options in the Tools menu, 40%, 40%]
+. Complete the *OAuth Token Creator* form that appears.
++
+image:generate-token-form.png[The OAuth Token Creator Form, 60%, 60%]
++
+If you already configured the Credentials for OAuth Action Step details and clicked the *Save data to generate Tokens via the Tools menu* button, the data automatically populates with the last data you entered in the wizard.
+. Click *Authenticate*.
+. Click *Copy Token to Clipboard*.
+. Paste the token in an alphanumeric Server Based Variable.
```

Comment:
Rename 'Server Based Variable' to 'Activity Parameter'.

Server Based Variable is a very old term from Servicetrace, no longer valid in MS RPA.